x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -15,7 +15,20 @@ import {
   dataAnalyticsExplainSchema,
 } from '../new_platform/data_analytics_schema';
 
+/**
+ * Routes for the data frame analytics
+ */
 export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteInitialization) {
+  /**
+   * @apiGroup DataFrameAnalytics
```

Comment:
Does this have to be a single word? For example `Data frame analytics API` would make a better navigation title in the docs.

Reply:
It's possible to provide a sentence, but the result output will be `Data_frame_analytics_API. Discussed and decided to stick to the current naming.